README.md

```diff
@@ -58,7 +58,7 @@ docker-compose up -d postgis
 docker-compose run --rm \
   -e osmupdate_extra_params="--base-url=download.geofabrik.de/europe/switzerland-updates/" \
   -e osm_planet_mirror="http://download.geofabrik.de/" \
-  -e osm_planet_path_relative_to_mirror="europe/switzerland-160321.osm.pbf" \
+  -e osm_planet_path_relative_to_mirror="europe/switzerland-160301.osm.pbf" \
```

Comment:
Why are these files ["more likely to be available"](https://github.com/geometalab/osm-world-db/pull/5/commits/b10cf26be21a7d47bdfa7bfac7981d1eafb86788)?